addon/modifiers/scroll-into-view.js

```diff
@@ -0,0 +1,22 @@
+import { modifier } from 'ember-modifier';
+
+export default modifier(function scrollIntoView(
+  element,
+  positional,
+  named = {}
+) {
+  const { options, shouldScroll } = named;
+  let hasBeenRemoved;
+
+  const trustedPromise = Promise.resolve(shouldScroll);
```

Comment:
not sure behind the meaning of `trustedPromise`, why name like this?

Reply:
How does `shouldScrollPromise` sound?